pkg/lib/sets/strset/strset.go

```diff
@@ -37,6 +37,10 @@ func New(ts ...string) Set {
 	return s
 }
 
+func FromSlice(items []string) Set {
```

Comment:
This one is called a few times in `cli/local/api.go`. It is used because in the context of those utilizations, going with `New(items...)` would become confusing to the reader.